x-pack/legacy/plugins/monitoring/server/kibana_monitoring/collectors/ops_buffer/ops_buffer.js

```diff
@@ -26,7 +26,7 @@ export function opsBuffer(server) {
   return {
     push(event) {
       eventRoller.addEvent(event);
-      server.log(['debug', LOGGING_TAG, KIBANA_MONITORING_LOGGING_TAG], 'Received Kibana Ops event data');
```

Comment:
Wondering if this debug helps you folks, or  you think it would be OK to remove. It generates a lot of noise when running in verbose.

Reply:
I think it would be okay. I don't know how useful it is to folks, but my gut says not that useful